Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.iOS/AppDelegate.cs

```diff
@@ -1,17 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Contoso.Forms.Demo.iOS;
 using Foundation;
 using Microsoft.AppCenter;
 using Microsoft.AppCenter.Analytics.iOS.Bindings;
 using Microsoft.AppCenter.Distribute;
 using UIKit;
+using Xamarin.Forms;
 
+[assembly: Dependency(typeof(AppDelegate))]
 namespace Contoso.Forms.Puppet.iOS
 {
     [Register("AppDelegate")]
-    public class AppDelegate : Xamarin.Forms.Platform.iOS.FormsApplicationDelegate
+    public class AppDelegate : Xamarin.Forms.Platform.iOS.FormsApplicationDelegate, IClearCrashClick
     {
+        private const string CrashesUserConfirmationStorageKey = "MSUserConfirmation";
```

Comment:
This does not compile.
Please test locally before pushing changes.

Reply:
Same for Android in MainActivity.cs for the Forms.Puppet version.

Reply:
Fix this issue and test locally in a new commit.